.github/workflows/build.yml

```diff
@@ -59,6 +59,11 @@ jobs:
         with:
           node-version-file: .node-version
 
+      - name: Install libvips
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libvips
```

Comment:
We don't actually use libvips yet. Maybe we should switch to libvips in the current Rails version first.